rust-src/key_derivation/src/lib.rs

```diff
@@ -257,6 +258,10 @@ impl ConcordiumHdWallet {
         ])?;
         let attribute_commitment_randomness_seed =
             derive_from_parsed_path(&path, &self.seed)?.private_key;
+        println!(
+            "attribute_commitment_randomness_seed: {:?}",
+            attribute_commitment_randomness_seed
+        );
```

Comment:
Not sure where this got in, but I don't think it should stay.

Reply:
Removed here in in other places, where it should not be.